enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -16,113 +17,223 @@ import {
-    whitelabel: {
-      name: t`Appearance`,
-      settings: [
-        {
-          key: "application-name",
-          display_name: t`Application Name`,
-          type: "string",
-        },
-        {
-          key: "application-font",
-          display_name: t`Font`,
-          widget: FontWidget,
-        },
-        {
-          key: "application-font-files",
-          widget: FontFilesWidget,
-          getHidden: settings => settings["application-font-files"] == null,
-        },
-        {
-          key: "application-colors",
-          display_name: t`Color Palette`,
-          widget: ColorSettingsWidget,
-        },
-        {
-          key: "application-logo-url",
-          display_name: t`Logo`,
-          type: "string",
-          widget: LogoUpload,
-        },
-        {
-          key: "application-favicon-url",
-          display_name: t`Favicon`,
-          type: "string",
-        },
-        {
-          key: "landing-page",
-          display_name: t`Landing Page`,
-          type: "string",
-          placeholder: "/",
-          widget: LandingPageWidget,
-        },
-        {
-          key: "loading-message",
-          display_name: t`Loading message`,
-          type: "select",
-          options: getLoadingMessageOptions(),
-          defaultValue: "doing-science",
-        },
-        {
-          key: "help-link",
-          display_name: t`Help Link in the Settings menu`,
-          description: (
-            <p>
-              {t`The Settings menu includes a Help link that goes to `}
-              <Anchor href="https://www.metabase.com/help">{t`this page`}</Anchor>
-              {t` by default.`}
-            </p>
-          ),
-          widget: HelpLinkSettings,
-          defaultValue: "metabase",
-        },
-        {
-          key: "show-metabot",
-          display_name: t`Metabot greeting`,
-          description: null,
-          type: "boolean",
-          widget: MetabotToggleWidget,
-          defaultValue: true,
-        },
-        {
-          key: "show-lighthouse-illustration",
-          display_name: t`Lighthouse illustration`,
-          description: null,
-          type: "boolean",
-          widget: LighthouseToggleWidget,
-          defaultValue: true,
-        },
-        {
-          key: "show-metabase-links",
-          display_name: t`Documentation and references`,
-          description: <MetabaseLinksToggleDescription />,
-          widget: MetabaseLinksToggleWidget,
-        },
-      ],
+  PLUGIN_ADMIN_SETTINGS_UPDATES.push(
+    sections => ({
+      ...sections,
+      whitelabel: {
+        name: t`Appearance`,
+        key: "-redirect-to-branding",
+        settings: [
+          {
+            key: "-redirect-to-branding",
+            widget: () => (
+              <RedirectWidget to="/admin/settings/whitelabel/branding" />
+            ),
+          },
+        ],
+      },
+      "whitelabel/branding": {
+        tabs: [
```

Comment:
I'm not totally satisfied with the current API. I initially wanted the shape to be like
```json
{
  "tabs": [{
    "name": "tab1",
    "settings": []
  }, {
    "name": "tab2",
    "settings": []
  }]
}
```

but it turns out we expect the `settings` to always be there and be an array, so without changing a lot more components I introduce the new `tabs` key and specify which setting in the `settings` to display in which tabs instead.

Actually if we don't make each tab having their own URL then it would be more consistent. In this case we need to re-specify the same `tabs` again.